common/presenters/message-banner/assessment-to-handed-over-banner.js

```diff
@@ -33,6 +33,17 @@ module.exports = function assessmentToHandedOverBanner({
           context,
         })}
       </a>
+
+      <p class="app-icon app-print--hide">
+        ${i18n.t('messages::assessment.handed_over.note', {
+          context,
```

Comment:
You can skip passing `context` and `details` in as they're not used by the translation.